src/libcore/iter/traits/iterator.rs

```diff
@@ -242,6 +235,8 @@ pub trait Iterator {
     /// let a = [1, 2, 3, 4, 5];
     /// assert_eq!(a.iter().count(), 5);
     /// ```
+    ///
+    /// [`usize::MAX`]: crate::usize::MAX
```

Comment:
nit: maybe move this to line 227 where it was on originally?

Reply:
Done.